src/builds/water.css

```diff
@@ -0,0 +1,9 @@
+/**
+ * Automatic version:
+ * Uses light theme by default but switches to dark theme
+ * if a system-wide theme preference is set on the user's device.
+ */
+
+@import '../variables-light.css';
```

Comment:
Here, it looks like light is the default, but our docs say that dark is the default. This build file should be changed so that we have:
```
@import '../variables-dark.css';
@import '../variables-light.css'  (prefers-color-scheme: light);
```

Reply:
Nice catch! Fixed in f7a0b68f08b4bc2a75c4bf8d24057fce15303ec0

Reply:
The docs site needs to be updated as well then, it says Light is the default in the theme section of the info table.

Reply:
Oof. In 476074e62eb823933189eb303be5fa4af171a0c5 I reverted the file, and fixed the readme to say that light theme is the default. Anything else I missed?

Reply:
Ah, so light is now the default everywhere? I'm very much in favor of this, it better matches the "vanilla" default of the web and thus general user expectations and also seems to be [what browsers (at least Firefox) are doing](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme):

> If you have set privacy.resistFingerprinting to true, prefers-color-scheme preference is overridden to light.

But I think @kylejrp advocated for the dark theme as our opinionated default in another discussion, so let's hear what he has to say here :)

Reply:
I think my preference is for the default to be dark because the original water.css was released as only a dark theme. Keeps it nice for users upgrading, then the path can be from `water.css@1 -> water.css@2` with their site being dark by default, instead of `water.css@1 -> dark.css@2` and their site is forced to always be dark.

If we're planning on switching to light by default for `water.css`, we should probably mention that in the release notes as part of the migration for current users.

However, more than 80% of users have browsers that support `prefers-color-scheme`, and I think that it won't matter too much in the end either way. 😄

Reply:
@kognise not sure if this is resolved, but there needs to be a decision on if we're using light or dark by default and this should be documented in the changelog